motr/init.c

```diff
@@ -188,6 +193,9 @@ struct init_fini_call subsystem[] = {
 	{ &m0_net_lnet_init,    &m0_net_lnet_fini,    "net/lnet" },
 	{ &m0_mem_xprt_init,    &m0_mem_xprt_fini,    "bulk/mem" },
 #ifndef __KERNEL__
+#ifdef ENABLE_LIBFAB
+	{ &m0_net_libfab_init,   &m0_net_libfab_fini,   "net/libfab" },
+#endif /* ENABLE_LIBFAB */
```

Comment:
If !ENABLE_LIBFAB, libfab.[ch] should define m0_net_libfab_{init,fini}() as no-op functions.

Reply:
Done.